### PR TITLE
feature: dummy project

### PR DESCRIPTION
**Pull request by OdunayoOkebunmi**
### What does this PR do?
connects #15

### CODE REVIEW

File: src/screens/App/screens/Main/screens/Projects/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import Post from './Post';
+import Posts from './Posts';
+
+export default (
+  <Route name="main">
+    {Posts}
```
Comment (frknasir) [inline src/screens/App/screens/Main/screens/Projects/index.js]: here it looks like you still have post routes and in a projects related file

File: src/screens/App/screens/Main/screens/Blog/components/PostsCard.js [hunk 2]
```diff
@@ -1,32 +1,31 @@
 /* eslint-disable react/jsx-props-no-spreading */
 import React, { useContext } from 'react';
 import BlogContext from '@Context/blog/BlogContext';
-import PostCard from './PostCard';
-import Pagination from './Pagination';
+import Pagination from '@App/shared/components/pagination/Pagination';
+import PostCard from '@SharedComponents/PostCard/PostCard';
```
Comment (frknasir) [inline src/screens/App/screens/Main/screens/Blog/components/PostsCard.js]: can PostCard and Pagination folder have an index file that will be the default?

File: src/config/routes/index.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// import routes
```
Comment (frknasir) [inline src/config/routes/index.js]: I dont think we really need this file anymore...

File: src/screens/App/screens/Main/screens/Blog/Post/index.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Route } from 'react-router-dom';
-import Post from './Post';
+import SinglePageComponent from '@SharedComponents/PostCard/SinglePageComponent';
```
Comment (frknasir) [inline src/screens/App/screens/Main/screens/Blog/Post/index.js]: I feel like we should let Post be Post and probably the components that should be shared with projects are `card` and `pagination`...what do you think?

File: src/screens/App/screens/Main/screens/Projects/Post/index.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```
Comment (frknasir) [inline src/screens/App/screens/Main/screens/Projects/Post/index.js]: lets leave this component as it is and only for blog and not try to abstract

File: src/screens/App/screens/Main/screens/Projects/Posts/Posts.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ProjectsPage from '../components/ProjectsPage';
+
+const Posts = () => <ProjectsPage />;
```
Comment (frknasir) [inline src/screens/App/screens/Main/screens/Projects/Posts/Posts.js]: its not clear what this is doing....

File: src/screens/App/screens/Main/screens/Projects/Posts/index.js [hunk 7]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```
Comment (frknasir) [inline src/screens/App/screens/Main/screens/Projects/Posts/index.js]: the folder structure here is offfff

File: src/screens/App/screens/Main/screens/Projects/components/ProjectCategories.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```
Comment (frknasir) [inline src/screens/App/screens/Main/screens/Projects/components/ProjectCategories.js]: I know there isnt a design in place...but  for inspiration, we should look at this https://udeme.ng/project/?projectId=0YaX7goOkm55

Comment (OdunayoOkebunmi) [reply src/screens/App/screens/Main/screens/Projects/components/ProjectCategories.js]: Noted

**Pull request merged**